Postman/Survey/SurveyPublicIds.postman_collection.json

```diff
@@ -90,7 +105,7 @@
 				],
 				"body": {
 					"mode": "raw",
-					"raw": "[\r\n  {\r\n    \"Id\": \"sample string 1\",\r\n    \"LinkType\": \"sample string 2\",\r\n    \"Active\": true,\r\n    \"Url\": \"sample string 4\"\r\n  },\r\n  {\r\n    \"Id\": \"sample string 1\",\r\n    \"LinkType\": \"sample string 2\",\r\n    \"Active\": true,\r\n    \"Url\": \"sample string 4\"\r\n  }\r\n]"
+					"raw": "[\r\n  {\r\n    \"Id\": \"{{LinkPublicId}}\",\r\n    \"LinkType\": \"LiveId\",\r\n    \"Active\": false,\r\n    \"Url\": \"https://yellow-interviewing.niposoftware-dev.com/Interviews/CoReV/L6YjSMOwTBPzYUwMNggr\"\r\n  }\r\n]"
```

Comment:
This is a specific survey live link pointing to our test server, I don't think it belongs here.